cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1324,7 +1324,9 @@ def _add_launch_templates(self, managed_placement_groups, instance_profiles):
 
             queue_placement_group = None
             if queue.networking.placement_group:
-                if queue.networking.placement_group.id:
+                if queue.networking.placement_group.id and (
```

Comment:
Could you kindly add a comment on why we're checking the `enabled` property of the PlacementGroup here?
If a customer defines the PlacementGroup Name and it's been validated, we can go ahead and use it, right?

Reply:
In the cluster_stack.py, I add the check there because in case customers skip the Validators, and set `PlacementGroup/Id` and `PlacementGroup/Enabled` is explicitly set to `false`, the placementgroup id will not be used

Reply:
I see, that makes sense, Thanks!